build.sbt

```diff
@@ -23,39 +23,40 @@ lazy val core = mkModule("core")
   .settings(
     libraryDependencies ++= List(
       // TODO: replace Calvin's version with the canonical one once its released
-      "com.github.kaizen-solutions.trace4cats-newrelic" %% "trace4cats-newrelic-http-exporter" % "0.12.2",
+      trace4cats("newrelic-http-exporter"),
       "org.http4s" %% "http4s-async-http-client" % "0.23.10",
     )
   )
 
 lazy val sttp = mkModule("sttp")
   .settings(
     libraryDependencies ++= List(
-      "com.softwaremill.sttp.client3" %% "async-http-client-backend-zio1" % "3.6.1",
+      "com.softwaremill.sttp.client3" %% "async-http-client-backend-zio" % "3.5.2",
```

Comment:
Why go back from 3.6.1 -> 3.5.2?

Reply:
In a word/phrase: dependency hell.

The latest version of sttp uses a ZIO RC candidate that is incompatible with the ZIO RC version that zio-http, so we have to downgrade the version of sttp to get things to align here.

Reply:
😭